code/modules/mob/mob_defines.dm

```diff
@@ -193,7 +193,7 @@
 	var/move_on_shuttle = 1 // Can move on the shuttle.
 	var/captured = 0 //Functionally, should give the same effect as being buckled into a chair when true.
 
-	var/movement_speed_modifier = 1
+	var/movement_speed_modifier = 6
```

Comment:
I added the variable *specifically* so that it could be altered in order to make mobs move faster than the default speed.
However it makes no sense to have it altered by default. "1" is supposed to represent the current default speed so that, for example, you can make mobs move twice as fast by setting it to 2, or make them move half as fast by setting it to 0.5.
If you need to make mobs move faster in order to have your thing work, why not add the line `move_delay *= 6` just above line 305 in mob_movement.dm instead of altering this variable?